### PR TITLE
Feat(Tax Center) Tax Center page

### PR DESCRIPTION
**Pull request by hcondori-bc**
## Description (optional)
 - card report was added
 - card services was added
 - carousel with description was added

## Testing Steps (optional)

tax-center-page

### CODE REVIEW

File: packages/blockchain-info-components/index.d.ts [hunk 1]
```diff
@@ -277,3 +277,15 @@ export const CheckBoxInput: FunctionComponent<{
   onClick?: () => void
   onChange?: () => void
 }>
+export const SelectInput: FunctionComponent<{
+  disabled?: boolean,
+  elements?: Array<any>,
+  grouped?: boolean,
+  label?: string | any,
```
Comment (cmonti-bc) [inline packages/blockchain-info-components/index.d.ts:284]: is it ok to use `any` as type option? shouldn't we use `unknown` perhaps? [Here's the PR](https://github.com/Microsoft/TypeScript/pull/24439) for the v3 of typescript which includes this new safe typing counterpart.

File: packages/blockchain-wallet-v4-frontend/src/scenes/Help/index.tsx [hunk 2]
```diff
@@ -51,15 +52,7 @@ const IconTextRow = styled.div`
...
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  width: 48px;
-  height: 48px;
-  min-width: 48px;
-  background-color: ${(props) => props.theme.blue000};
-  border-radius: 48px;
+const StyledCircle = styled(CircleBackground)`
   margin: 8px 16px 8px 0;
```
Comment (cmonti-bc) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Help/index.tsx:56]: should we use pixels?

**Review by cmonti-bc — APPROVED** [review]

File: packages/blockchain-info-components/index.d.ts (hunk 1 above)
Comment (hcondori-bc) [reply packages/blockchain-info-components/index.d.ts:284]: yep, I think is better use `unknown`. thank you charly

File: packages/blockchain-wallet-v4-frontend/src/scenes/Help/index.tsx (hunk 2 above)
Comment (hcondori-bc) [reply packages/blockchain-wallet-v4-frontend/src/scenes/Help/index.tsx:56]: Saw pixel in everywhere, in this case just create a styled component with the margin that already had

**Review by cmonti-bc — APPROVED** [review]

**Review by julianromero-bc — APPROVED** [review]

**Review by plondon — APPROVED** [review]
LGTM

File: lang/en.json [hunk 3]
```diff
@@ -2087,6 +2087,23 @@
   "scenes.settings.tor.description2": "The Tor network is frequently used by hackers attempting to access Blockchain users wallets.",
   "scenes.settings.tor.title": "Wallet Access via Tor",
   "scenes.swap.low_fees": "Low Fees",
+  "scenes.tax.center.title": "Blockchain.com Tax Center",
```
[the remark this answers is not in the document]
Comment (hcondori-bc) [reply lang/en.json]: yes, removed

**Pull request merged**